---
layout: about
title: about
permalink: /
subtitle: Bioinformatician / Data Analyst | Expertise in Python, R, and Pipeline Development | Using Big Data to Understand Complex Datasets


profile:
  align: right
  image: prof_pic.jpg
  image_circular: true # crops the image to make it circular


selected_papers: true # includes a list of papers marked as "selected={true}"
social: true # includes social icons at the bottom of the page

announcements:
  enabled: false # includes a list of news items
  scrollable: true # adds a vertical scroll bar if there are more than 3 news items
  limit: 5 # leave blank to include all the news in the `_news` folder

latest_posts:
  enabled: false
  scrollable: true # adds a vertical scroll bar if there are more than 3 new posts items
  limit: 3 # leave blank to include all the blog posts
---

### Backround and Interests

I’m David Jacobson, PhD — a bioinformatician and data scientist passionate about transforming complex data into meaningful, data-driven insights.

At the Centers for Disease Control and Prevention (CDC), I led efforts in molecular surveillance of parasitic diseases, developing advanced analytical methods that reveal the genetic diversity of pathogens like Plasmodium. These insights directly informed outbreak response strategies and national surveillance efforts. By optimizing and containerizing bioinformatic workflows, our team has significantly improved the accuracy of genetic clustering and reduced turnaround times for critical public health investigations.

A core part of my role is translating complex molecular data into clear, actionable insights for a wide audience—including epidemiologists, laboratory scientists, and federal decision-makers. I bring a strong foundation in data science and analytics, with experience in statistical modeling, Power BI dashboarding, and integrating multi-source data in high-performance computing environments. My tools of choice include Python, R, Nextflow, Bash, Singularity, Docker, Git, and cloud/HPC platforms.

My background also includes a PhD in Anthropology, where I specialized in microbiome bioinformatics. I developed scalable workflows for analyzing shotgun metagenomic and 16S rRNA data, applying these tools to projects ranging from ovarian cancer and gut microbiome resilience to ancient DNA. I collaborated with clinicians, researchers, and international partners to generate rigorous, peer-reviewed publications that have expanded our understanding of microbial diversity across populations and contexts.

Whether through genomic surveillance, microbial ecology, or applied data analytics, I am dedicated to using computational tools to solve real-world problems. This portfolio highlights some of the impactful projects I’ve worked on and the skills I bring to the intersection of bioinformatics, data science, and public health.

### Navigating through the Portfolio

#### Projects
Jump to the [Publications tab]() to take a look at the various analyses, workflows, and reports that I've generated over the past few years. Note that the reports section consists of facsimiles of the reports I developed at CDC, but use dummy data instead of real outbreak clusters and IDs.

#### Publications
A list of of all publications where I am a first, last, or co-author can be found in the [Publications tab](). Alternatively, click the google scholor icon on the bottom of this page to be taken to my google scholar page.

#### CV / Resume
Links to my CV and resumes can be found in the [CV / Resume tab]()

#### Code Availability
Links to the code used to generate the content in the Projects tab can be found in the [repositories tab]()

